CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.7.0 (Unreleased)
+--------------------------
+
+- Add Black formatter support.
```

Comment:
- `Adds` keeping the usual style. ("This release does this and this, adds that and that, removes that and that, etc".) See past commit messages.

- For the git commit headline, we can just use `docs: ...` instead of `changes: ...`, in order to keep the number of various used headline prefixes small.

BTW we seem to have quite a disparity already, e.g. in this repo:

```console
$ git log --format=oneline | awk '{print $2}' | sort -u
api:
api_client:
changes:
config:
docker:
dockerfile:
docs:
global:
Initial
installation:
job-submission:
logging:
openapi:
proxy:
publisher:
release:
serial:
setup:
task:
tasks:
tests:
travis:
utils:
version:
```